<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单词记忆</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#label_10{ 
    color: red ;
    font-size:15px;
}
QLabel#label_11{ 
    color: red ;
    font-size:15px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>书写</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,9">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>假名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="JM1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>日语：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RY1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>中文：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ZW1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>声调：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SD1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="Button1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>验证</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>对照表</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>假名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="JM2">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>日语：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RY2">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>中文：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ZW2">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>声调：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SD2">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QPushButton" name="Button2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>下一个</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
